core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -182,6 +183,27 @@ class LogCleaner(initialConfig: CleanerConfig,
     cleanerManager.removeMetrics()
   }
 
+  /**
+   * Activate metrics
+   */
+  def activateMetrics():Unit = {
+    metricsGroup.newGauge(MaxBufferUtilizationPercentMetricName,
```

Comment:
Perhaps we can remove the declarations in the class field above around line 130?

Reply:
agree. @chiacyu Could you Please avoid producing duplicate code